conda-forge.yml

```diff
@@ -1,6 +1,5 @@
 bot:
   abi_migration_branches:
-  - v1.74.x
```

Comment:
Since this is going into `main` and we just moved to 1.78 ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3599 ), going ahead and dropping `v1.74.x` from ABI migration here for simplicity